templates/kapeta/block-type-frontend/ai-templates/ApplicationScreen.tsx.hbs

```diff
@@ -23,7 +23,7 @@ export const ApplicationScreen = () => {
         ];
     }, []);
 
-    {{#raw}}
+    {{{{raw}}}}
```

Comment:
Is escaping `{` necessary now? If not can we replace all `&#123;\{` with `{{` ?

Reply:
Yes, actually that's definitely needed. I forgot to push it.